collect_app/src/main/java/org/odk/collect/android/preferences/ServerPreferencesFragment.java

```diff
@@ -243,19 +244,28 @@ public void addOtherPreferences() {
     }
 
     public void initAccountPreferences() {
+        allowClickSelectedGoogleAccountPreference = true;
```

Comment:
You don't need this line. You can add `= true` in line 88

Reply:
Okay

Reply:
fixing those

Reply:
I think i did it well this time